build.gradle

```diff
@@ -14,6 +14,7 @@ repositories {
 dependencies {
     compile "org.apache.commons:commons-csv:1.8"
     compile "org.apache.poi:poi-ooxml:4.1.2"
+    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.10.4"
```

Comment:
If you pull my `update_deps` branch into this PR (or more likely pull `develop` into here after that branch is approved and merged into it), could you please make sure to update this number to the version that's in there for all the other jackson libraries?